src/io/flutter/editor/ExpressionParsingUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.editor;
+
+import java.awt.*;
+
+public class ExpressionParsingUtils {
```

Comment:
Could add some tests for this class.

Reply:
Filed bug https://github.com/flutter/flutter-intellij/issues/4145 to track this.